cmd/server/customers.go

```diff
@@ -183,8 +186,16 @@ func validateMetadata(meta map[string]string) error {
 }
 
 func (req customerRequest) asCustomer(storage *ssnStorage) (*client.Customer, *SSN, error) {
+	if req.CustomerID == "" {
+		req.CustomerID = base.ID()
+	}
+
+	if req.Status == "" {
+		req.Status = client.UNKNOWN
+	}
```

Comment:
We should not allow the Status to be set. It should always default to `UNKNOWN` at first.

Reply:
Similar to the `CustomerID` field above, since I'm reusing this object to update a `customer`, I don't want to overwrite the customer's status with `unknown`. 

To make sure we always set the default to `unknown`, I can set it in the in the repo's `createCustomer` method:

```go
...
	_, err = stmt.Exec(c.CustomerID, c.FirstName, c.MiddleName, c.LastName, c.NickName, c.Suffix, c.Type, c.BirthDate, client.UNKNOWN, c.Email, now, now, namespace)
...
```